### ISSUE TITLE
UnicodeDecodeError: 'utf8' codec can't decode bytes in position 31-33: invalid data

### ISSUE THREAD

**Issue by bbigras** [fbzhong/sublime-closure-linter#3]
I have an UnicodeDecodeError when I try to do "Run Closure Linter (ctrl+shift+j)".

```
Writing file /C/Dev/Python/Tests/test-canvas2/testcanvas/testcanvas/static/test.js with encoding UTF-8
Traceback (most recent call last):
  File ".\gjslint.py", line 64, in append_data
  File ".\encodings\utf_8.py", line 16, in decode
```

gjslint works fine from the command line

```
C:\Dev\Python\Tests\test-canvas2\testcanvas\testcanvas\static>gjslint test.js
1 files checked, no errors found.
```

test.js

```
$(document).ready(function() {
});
```

closure-linter==2.3.5
